### ISSUE TITLE
NVDA fails to read the statusbar in Thunderbird beta

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#4144]
**Reported by k_kolev1985 on 2014-05-16 08:22**
Due to some accessibility fixes related to the high-contrast mode in Windows 8, I've upgraded Mozilla Thunderbird to version 29.0 and now - to 30.0 (beta releases). Since then, I've noticed that NVDA will not read the statusbar in Thunderbird, even though the statusbar itself is shown on the screen. Instead, when I press NVDA+END, NVDA will say only "blank". This was not the case with the stable version of Thunderbird (v24.x) - there the statusbar was read by NVDA just fine.

Steps to reproduce it:
1. Upgrade Thunderbird to version 29 or 30 beta.
2. Launch Thunderbird.
3. From the "View" menu, from the "Toolbar" submenu, make sure that "Statusbar" option is checked, so the statusbar is shown on the screen.
4. Press NVDA+END to make NVDA read the content of the statusbar.

Actual results:
NVDA says only "blank".

Expected results:
NVDA should read the content of the statusbar.

Test environment:
- NVDA version: next-10598,0b6f13a; 2014.1.
- Mozilla Thunderbird version: 29.0 and 30.0 (both beta releases).
  **Blocked by #2697, #4327**

**Comment by ehollig** [comment]
Acording to [MozillaBug:934875](https://bugzilla.mozilla.org/show_bug.cgi?id=934875), this was fixed in Thunderbird 34.0.
